integ/Cargo.toml

```diff
@@ -6,11 +6,11 @@ edition = "2018"
 publish = false
 
 [dependencies]
-aws-config = "0.0.26-alpha"
-aws-sdk-ec2 = "0.0.26-alpha"
-aws-sdk-eks = "0.0.26-alpha"
-aws-sdk-iam = "0.0.26-alpha"
-aws-sdk-ssm = "0.0.26-alpha"
+aws-config = "0.8"
```

Comment:
The latest version is 0.10.1, is there anything blocking us from updating to the latest version? Otherwise we will be bugged by the bot again.

Reply:
I'm pretty confident with `0.8` and I haven't test with `0.10.1` version. I'll try with the latest version. : )